### DISCUSSION TITLE
Azure server-wide backups under empty folder

### DISCUSSION THREAD

**Discussion by lahma** [ravendb/ravendb#12284] [Q&A] [answered]
After changing backup to Azure configuration (preferred mentor node) with RavenDB server 5.1.7 I now see that our backups are ending under empty folder name, like below:

![image](https://user-images.githubusercontent.com/171892/119928714-cd8b9f80-bf84-11eb-9ae9-83a304acefff.png)

It used to be:

* backups
    * database1
    * database2

And now it's 

* backups
    *  "empty"
             * database1
             * database2

We haven't configured any remote folder name:

![image](https://user-images.githubusercontent.com/171892/119928796-f6139980-bf84-11eb-8bbf-a14eb903090f.png)

So is this a known issue? We recently upgraded from 4.2 series and just now updated to backup configuration using 5.1.7.

**Comment by ppekrol** [comment, marked as answer]
I've moved this to our bugtracker. We will take a look. Link: https://issues.hibernatingrhinos.com/issue/RavenDB-16777

**Reply by garayx** [reply]
@lahma can you send us the periodic backup configuration to support email (support@ravendb.net).

You can find the configuration in studio under Settings -> DatabaseRecord
![image](https://user-images.githubusercontent.com/16625387/120448441-3e232980-c394-11eb-836a-d6c4631e5b68.png)

**Reply by lahma** [reply]
@garayx  Email on its way.
